doc/bundles.md

```diff
@@ -0,0 +1,36 @@
+## Bundles
```

Comment:
Can we pull this out into it's own commit, or it's own PR entirely?

Reply:
Yes, sounds good, it should be in another PR altogether. I had some other changes locally that made their way through by accident